taxonomies/additives.txt

```diff
@@ -465,7 +483,8 @@ efsa_evaluation_url:en: http://dx.doi.org/10.2903/j.efsa.2015.4288
 efsa_evaluation_date:en: 2015-11-18
 efsa_evaluation:en: Scientific Opinion on the re-evaluation of cochineal, carminic acid, carmines (E 120) as a food additive
 additives_classes:en: en:colour
-mandatory_additive_class:en: en:colour
```

Comment:
I'll add this back, we need it to make sure that "carmine" (some kind of salad) does not trigger E120